mail_compose_select_lang/models/report.py

```diff
@@ -0,0 +1,18 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+# Copyright 2015 Pedro M. Baeza
+# For license notices, see __openerp__.py file in root directory
+##############################################################################
+from openerp import models, api
+
+
+class Report(models.Model):
+    _inherit = 'report'
+
+    @api.model
+    def translate_doc(self, doc_id, model, lang_field, template, values):
```

Comment:
Did yopu test how this affect to the global translate_doc, for this feature are you overwritting entirely the qweb translate system.

A test for that should be good (copying original ones).